src/views/StudyInfoView.js

```diff
@@ -71,12 +71,42 @@ const StudyInfoView = ({match, history}) => {
         />
       </Container>
     );
+  if (studyByKfId === null) {
+    return (
+      <>
+        <Helmet>
```

Comment:
Is it possible to move the `Helmet` titling into the `ErrorView` so you don't need to take care of it for each case?